Content.Goobstation.Shared/SlaughterDemon/BloodCrawlComponent.cs

```diff
@@ -0,0 +1,70 @@
+// SPDX-FileCopyrightText: 2025 GoobBot <uristmchands@proton.me>
+// SPDX-FileCopyrightText: 2025 Lumminal <81829924+Lumminal@users.noreply.github.com>
+//
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+using Content.Shared.Chemistry.Reagent;
+using Content.Shared.Polymorph;
+using Robust.Shared.Audio;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+
+namespace Content.Goobstation.Shared.SlaughterDemon;
+
+[RegisterComponent, NetworkedComponent]
+public sealed partial class BloodCrawlComponent : Component
+{
+    /// <summary>
+    /// This is the search range of the blood puddles
+    /// </summary>
+    [DataField]
+    public float SearchRange = 0.1f;
+
+    /// <summary>
+    /// This is the entity storing the action of this ability
+    /// </summary>
+    [DataField]
```

Comment:
```suggestion
    [ViewVariables]
```
If you aren't going to set it in YAML, there's no point in making it a datafield.